agentMET4FOF/agents/signal_agents.py

```diff
@@ -44,3 +50,86 @@ def agent_loop(self):
         if self.current_state == "Running":
             sine_data = self._sine_stream.next_sample()  # dictionary
             self.send_output(sine_data["quantities"])
+
+
+class StaticSineGeneratorWithJitterAgent(AgentMET4FOF):
+    """An agent streaming a pre generated sine signal of fixed length with jitter
+
+    Takes samples from the :py:mod:`StaticSineGeneratorWithJitter` and pushes them
+    sample by sample to connected agents via its output channel.
+    """
```

Comment:
Do we need to mention the parameters to display them on RTD?

Reply:
No, [we don't](https://agentmet4fof.readthedocs.io/en/sensor_issue_agents/agentMET4FOF_agents.html#agentMET4FOF.agents.signal_agents.StaticSineWithJitterGeneratorAgent).